src/mpid/ch4/shm/posix/eager/iqueue/iqueue_send.h

```diff
@@ -24,35 +24,35 @@ MPL_STATIC_INLINE_PREFIX size_t MPIDI_POSIX_eager_buf_limit(void)
  * and retry.
  *
  * grank   - The global rank (the rank in MPI_COMM_WORLD) of the receiving process.
+ * buf, count, datatype - Data buffer and signature for the send buffer
+ * am_hdr, am_hdr_sz    - am header
```

Comment:
We should mention that am_hdr is allowed to be `NULL`.

Reply:
I think you mean buffer is allowed to be NULL. am_hdr should not be NULL.